forge-gui/res/cardsfolder/upcoming/prayer_of_binding.txt

```diff
@@ -0,0 +1,9 @@
+Name:Prayer of Binding
+ManaCost:3 W
+Types:Enchantment
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile target nonland permanent an opponent controls until CARDNAME leaves the battlefield.
```

Comment:
Does this line need "You gain 2 life." at the end to show up in card detail?

Reply:
Good point , added !